cmd/messaging-tool/receive.go

```diff
@@ -0,0 +1,79 @@
+/*
+Copyright (c) 2018 Red Hat, Inc.
+
+Licensed under the Apache License, Version 2.0 (the "License");
+you may not use this file except in compliance with the License.
+You may obtain a copy of the License at
+
+  http://www.apache.org/licenses/LICENSE-2.0
+
+Unless required by applicable law or agreed to in writing, software
+distributed under the License is distributed on an "AS IS" BASIS,
+WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+See the License for the specific language governing permissions and
+limitations under the License.
+*/
+
+package main
+
+import (
+	"github.com/spf13/cobra"
+	"github.com/golang/glog"
+
+	"github.com/container-mgmt/messaging-library/pkg/client"
+)
+
+var receiveCmd = &cobra.Command{
+	Use:   "receive",
+	Short: "Receive messages from a destination",
+	Long:  "Receive messages from a destination.",
+	Run:   runReceive,
+}
+
+func runReceive(cmd *cobra.Command, args []string) {
+	// Check mandatory arguments:
+	if destinationName == "" {
+		glog.Errorf("The argument 'destination' is mandatory")
+		return
+	}
+
+	c := &client.StompConnection{
```

Comment:
There should be no mention of STOMP here either, but I understand that isn't possible at the moment.